app/services/file_workflow_service.py

```
@@ -511,247 +526,189 @@ def _process_single_file(
-                        file_size = original_stat.st_size
-                except (OSError, FileNotFoundError) as e:
-                    # Rollback status
-                    inventory_entry.status = old_status
-                    db.commit()
-                    result["error"] = f"Cannot stat source file: {e}"
-                    return result
-
-                # Progress tracking
-                file_name = file_path.name
+                backend = get_backend(storage_location)
+            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)